types/express-serve-static-core/index.d.ts

```diff
@@ -75,7 +75,7 @@ export type ErrorRequestHandler<
     req: Request<P, ResBody, ReqBody, ReqQuery, LocalsObj>,
     res: Response<ResBody, LocalsObj>,
     next: NextFunction,
-) => void | Promise<void>;
+) => void;
```

Comment:
This file contains the meaningful change, the rest are changes to put other package's tests back to where they were before they were updated to pass w/ the new return type